src/test/java/elrond/esdt/ESDTFunctionsTest.java

```diff
@@ -50,16 +50,16 @@ public void shouldExtractESDTTransferTypes() throws Exceptions.AddressException,
 
         Address sender = Address.fromHex("fd691bb5e85d102687d81079dffce842d4dc328276d2d4c60d8fd1c3433c3293");
         Address receiver = Address.fromHex("c70cf50b238372fffaf7b7c5723b06b57859d424a2da621bcc1b2f317543aa36");
-        String tokenIdentifier = "tkn";
+        String tokenIdentifierHex = "4552444a4156412d333866323439";
```

Comment:
```suggestion
        String tokenIdentifier = "ERDJAVA-38f249";
        String tokenIdentifierHex = Utils.castToPaddedHex(tokenIdentifier); 
```
if you`ll add the castToPaddedHex also for String